packages/help-center/src/hooks/use-get-history-chats.ts

```diff
@@ -73,8 +72,8 @@ export const useGetHistoryChats = (): UseGetHistoryChatsResult => {
 			...conversationsWithUpdatedStatuses,
 			...filteredOdieConversations,
 		].sort( ( a, b ) => {
-			const createdAtA = getConversationCreatedAt( a ) ?? 0;
-			const createdAtB = getConversationCreatedAt( b ) ?? 0;
+			const createdAtA = Number( a.messages?.[ 0 ]?.received ) * 1000 || 0;
+			const createdAtB = Number( b.messages?.[ 0 ]?.received ) * 1000 || 0;
```

Comment:
Shouldn't we use `getLastMessage()` to access the last message? That's the message that will be shown in the `History` panel and the `Recent Conversation` section. As far I can see, this code will retrieve the first message of a conversation and not the last one.

Reply:
Also, are you sure we should delete the `getConversationCreatedAt()` helper function? It seems it would allow us to simplify the code a bit, and remove duplication (assuming we would rename it to `getConversationLastMessageReceivedAt()`). Another option could be to locate that function above that hook, if you think we may not need it in other parts of our code.